packages/swapper/src/swappers/thorchain/utils/estimateTradeFee/estimateTradeFee.ts

```diff
@@ -59,40 +59,41 @@ export const estimateTradeFee = async (
     })
 
   const gasRate = inboundInfo.gas_rate
-  const { chainId, assetNamespace } = fromAssetId(buyAsset.assetId)
+  const { chainId: buyChainId, assetNamespace } = fromAssetId(buyAsset.assetId)
 
-  const feeAssetId = getFeeAssetIdFromAssetId(buyAsset.assetId)
-  if (!feeAssetId)
+  const buyFeeAssetId = getFeeAssetIdFromAssetId(buyAsset.assetId)
```

Comment:
given we're trying to kill this function here https://github.com/shapeshift/lib/pull/908

and we (likely?) have the chain adapter in the deps to be able to get the fee asset, any chance you could quickly switch this over?